README.md

```diff
@@ -1,191 +1,58 @@
-# Ably Chat SDK for Swift
+![Ably Chat Swift Header](images/SwiftChatSDK-github.png)
+![CocoaPods](https://img.shields.io/cocoapods/v/Ably.svg)
```

Comment:
These two links refer to ably-cocoa; also, we don't support CocoaPods in this ably-chat-swift.

Reply:
We have the specific one now (https://swiftpackageindex.com/ably/ably-chat-swift) but the problem is that our non-macOS builds on Swift Package Index are failing, meaning that the platform compatibility badge only shows macOS. I've created #302 for this. Up to you whether you'd prefer to put the badges in or wait until we've fixed that (it's probably not something we'll fix right away).